lib/validator.js

```diff
@@ -215,6 +244,25 @@ async function validateRawClientDataJson() {
 	return true;
 }
 
+async function validateTransports() {
+	// XXX: this isn't very useful, since this has already been parsed...
```

Comment:
Can you explain this comment?

Reply:
Copy paste from another function. I removed it.